src/app/FunZone/simon/simon.component.scss

```diff
@@ -0,0 +1,46 @@
+.panel {
+  width: 150px;
+  height: 150px;
+  display: inline-block;
+  cursor: pointer;
+}
+
+#score {
+  text-align: left;
+  margin-right: 78rem;
+  width: 50px;
+  display: inline;
+}
+
+.background {
+  text-align: center;
+  background-color: #7e0c2b;
+}
+
+.panel:hover {
+  box-shadow: inset 0px 0px 0px 8px #7e0c2b;
+  box-sizing: border-box;
+}
+.bottom-right-panel {
+  border-bottom-right-radius: 80%;
```

Comment:
Maybe if this were 50% it would be a perfect circle?
Could look better.
Your choice